docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "2"
```

Comment:
As we spoke before, we should investigate and resolve an issue, why do we need to use `2` instead of `3` for Docker syntax version.